knime_extension/src/nodes/spatialnetwork.py

```diff
@@ -1423,3 +1423,284 @@ def execute(self, exec_context: knext.ExecutionContext, input1, input2):
+        "Isochrone time budget list",
+        """Input an interval list in minutes separated by comma e.g. 5,10,15,20,25,30""",
+        default_value="5,10,15,20,25,30",
+    )
+
+    depart_at = knext.DateTimeParameter(
+        "Departure time",
+        """The departure time for the isochrone calculation. This parameter can affect the isochrone map due to 
+        varying traffic conditions at different times.""",
+        default_value=None,
+        show_time=True,
+    )
+
+    traffic = knext.BoolParameter(
+        "Consider current traffic",
+        """If selected the current traffic conditions is considered in the isochrone calculation. 
+        Note that information on historic road speeds is always used.""",
+        default_value=True,
+    )
+
+    route_type = knext.EnumParameter(
+        "Route type",
+        """Determines the type of route used for the isochrone calculation. 
+        Different route types can result in different isochrones.""",
+        default_value=_TomTomRouteType.get_default().name,
+        enum=_TomTomRouteType,
+        style=knext.EnumParameter.Style.DROPDOWN,
+    )
+
+    travel_mode = knext.EnumParameter(
+        "Travel mode",
+        """Specifies the mode of travel for the isochrone calculation, which can 
+        significantly impact the shape and extent of the isochrone.""",
+        default_value=_TomTomTravelMode.get_default().name,
+        enum=_TomTomTravelMode,
+    )
+
+    tomtom_api_key = knext.StringParameter(
+        "TomTom API Key",
+        """The 
+        [TomTom API key](https://developer.tomtom.com/knowledgebase/platform/articles/how-to-get-an-tomtom-api-key/)
+        is required to authenticate requests to the 
+        [Calculate Reachable Range service](https://developer.tomtom.com/routing-api/documentation/routing/calculate-reachable-range)
+        which is part of the [Routing API](https://developer.tomtom.com/routing-api/documentation/routing/routing-service)
+        provided by TomTom. 
+        To get an API key, click
+        [here.](https://developer.tomtom.com/knowledgebase/platform/articles/how-to-get-an-tomtom-api-key/)
+        For details about the pricing go to the [TomTom pricing page.](https://developer.tomtom.com/store/maps-api)""",
+        default_value="your api key here",
+        validator=knut.api_key_validator,
+        # TODO: Validate that the API key is present
+    )
+
+    timeout = knext.IntParameter(
+        "Request timeout in seconds",
+        "The maximum time in seconds to wait for the request to the TomTom API to succeed.",
+        120,
+        min_value=1,
+        is_advanced=True,
+    )
+
+    _COL_GEOMETRY = "Geometry"
+    _COL_ISOCHRONE = "Time budget (Mins)"
+
+    def configure(self, configure_context, input_schema_1):
+        self.c_geo_col = knut.column_exists_or_preset(
+            configure_context, self.c_geo_col, input_schema_1, knut.is_geo
+        )
+
+        return knext.Schema(
```

Comment:
I need some help for this Schema. It still show the WARN, 
`DataSpec generated by configure does not match spec after execution.`
Can you help me understand what's wrong with my configuration?
@koettert